views/js/review/plugins/navigation/review-panel/tpl/stepOverview.tpl

```diff
@@ -0,0 +1,35 @@
+<ul class="step-overview-sections">
+    {{#each sections}}
+    <li class="step-overview-section">
+        {{#if ../displaySectionTitles}}
+            <span class="step-overview-section-label">
+                {{label}}
+            </span>
+        {{/if}}
+        <ul class="step-overview-items">
+            {{#each items}}
```

Comment:
I do slightly prefer the version in #99 where the section titles and outer list are excluded from this component.